Xwt.XamMac/Xwt.Mac/PopoverBackend.cs

```diff
@@ -203,7 +203,11 @@ public void DisableEvent (object eventId)
 		public void Show (Popover.Position orientation, Widget referenceWidget, Rectangle positionRect, Widget child)
 		{
 			var refBackend = Toolkit.GetBackend (referenceWidget) as IWidgetBackend;
-			NSView refView = (refBackend as ViewBackend)?.Widget;
+
+			NSView refView = (refBackend as EmbedNativeWidgetBackend)?.InnerView;
```

Comment:
Why is this required? Also it could break event handling, for events being attached after the widget has been added to the Xwt tree. I've got the feeling that we handle a product specific corner case here... and should make sure that we don't break other potential use cases.